config/database.yml

```diff
@@ -2,7 +2,7 @@
 
 <% user = ENV.key?("POSTGRESQL_ADMIN_PASSWORD") ? "root" : ENV["POSTGRESQL_USER"] %>
 <% password = ENV.key?("POSTGRESQL_ADMIN_PASSWORD") ? ENV["POSTGRESQL_ADMIN_PASSWORD"] : ENV["POSTGRESQL_PASSWORD"] %>
```

Comment:
not related to this PR, but why do we have 2 different env vars for the password?

we could make the lines above use `.fetch` too